packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.tsx

```diff
@@ -1,40 +1,46 @@
-import React, { useMemo, useState } from 'react';
+/* eslint-disable sort-keys */
+import React, { useState } from 'react';
 import { zxcvbn } from '@zxcvbn-ts/core';
+import { Score } from '@zxcvbn-ts/core/dist/types';
 import { WalletTextField } from '../WalletTextField';
 import { WalletTextFieldProps } from '../WalletTextField/WalletTextField';
 import PasswordMeter, { PasswordMeterProps } from './PasswordMeter';
 import PasswordViewerIcon from './PasswordViewerIcon';
 import './WalletPasswordField.scss';
 
-type StrengthMessage = Record<1 | 2 | 3 | 4, string>;
-
 interface WalletPasswordFieldProps extends WalletTextFieldProps, PasswordMeterProps {
-    messageObj?: StrengthMessage;
+    password: string;
     showPasswordMeter?: boolean;
 }
 
-const WalletPasswordField: React.FC<WalletPasswordFieldProps> = ({ messageObj, showPasswordMeter = true }) => {
-    const [password, setPassword] = useState('');
+const WalletPasswordField: React.FC<WalletPasswordFieldProps> = ({ password, showPasswordMeter = true, ...rest }) => {
```

Comment:
Avoid using true as default. Flip it for the better. like disablePasswordMeterfield or something

Reply:
I don't know what you mean by this 😅